crates/wash-cli/src/bin/wash.rs

```diff
@@ -230,6 +230,15 @@ async fn main() {
         .with_writer(std::io::stderr)
         .with_env_filter(EnvFilter::from_default_env())
         .init();
+
+    // TODO(pre-1.0): remove me
+    if let Ok(lattice) = std::env::var("WASMCLOUD_LATTICE_PREFIX") {
+        eprintln!(
+            "The `WASMCLOUD_LATTICE_PREFIX` environment variable is deprecated and will be removed. Please use `WASMCLOUD_LATTICE` instead."
+        );
+        std::env::set_var("WASMCLOUD_LATTICE", lattice);
```

Comment:
Could this end up munging something if they have both vars set? I don't think this is blocking, but I have something deep in my memory (that could be totally false) that setting env vars for people (unless that is the actual intent of the user) is bad practice

Reply:
I agree, but can you help me come up with an alternative approach? Based on my reading, clap only supports associating a single env var with an option using the macro :/

I'm okay with the grossness, given this is temporary ™️ code that should be released after host version n+1 is out. But if there's a way to avoid it entirely, I'm all ears